### PR TITLE
Update server setup and config documentation

### PR DESCRIPTION
**Pull request by hjkristenson**
We are moving to a 2-server installation, where one server is used for processing and the other hosts the image service. 
This PR is for the associated documentation changes in support of this new approach, and also reflects other updates to our workflow that have evolved since the initial documentation effort.

### CODE REVIEW

File: image_server/server_setup.md [hunk 1]
```diff
@@ -43,7 +68,7 @@ It takes about 5 minutes to stand up the instance and load balancer. The load ba
 
 ## Configure the Server
 
-1. Retrieve the appropriate prvc licence files from s3://hyp3-software/ and scp them to the server.
+1. Retrieve the appropriate prvc licence files for the ArcGIS Server and Image Server from s3://hyp3-software/ and scp them to the server
```
Comment (jacquelynsmale) [inline image_server/server_setup.md:71]:
```suggestion
1. Retrieve the appropriate prvc licence files on your local machine for the ArcGIS Server and Image Server from s3://hyp3-software/ and scp them to the server
```

File: image_server/server_setup.md [hunk 2]
```diff
@@ -70,11 +102,38 @@ export SITE_PASSWORD=<new password for the siteadmin user in the manager app>
...
+```
+
+8. Activate the arcpy conda environment and verify that the arcpy package can be imported
+```
+conda activate arcpy
+python
+import arcpy
+```
+   - the ArcGIS licensing must be valid to access the arcpy package
+   - if the arcpy package is not available for import, ensure that the ArcGIS Server license has been activated, and install arcpy in the arcpy conda environment once the license has been successfully applied
```
Comment (jacquelynsmale) [inline image_server/server_setup.md:120]:
```suggestion
   - the first-time import for arcpy might take a couple of minutes, but if the arcpy package is not available for import, ensure that the ArcGIS Server license has been activated, and install arcpy in the arcpy conda environment once the license has been successfully applied
```

**Review by jacquelynsmale — APPROVED** [review]

File: image_server/server_setup.md [hunk 3]
```diff
@@ -30,38 +53,50 @@ If setting up using the UI, go to CloudFormation and launch a new stack using ne
...
 It takes about 5 minutes to stand up the instance and load balancer. The load balancer will become more critical if we add additional servers.
 
-## Configure the Server
+## Configure the ArcGIS Server
+These steps are required for all servers, regardless of whether they will be configured as processing servers or image servers.
 
-1. Retrieve the appropriate prvc licence files from s3://hyp3-software/ and scp them to the server.
+1. Retrieve the appropriate prvc licence files on your local machine for the ArcGIS Server and Image Server from s3://hyp3-software/ and scp them to the server
 ```
 Aws s3 –profile hyp3 s3://hyp3-software/ . –recursive –exclude “*” –include “*.prvc”
```
Comment (asjohnston-asf) [inline image_server/server_setup.md:74]:
```suggestion
aws s3 cp --profile hyp3 s3://hyp3-software/ . --recursive --exclude "*" --include "*.prvc"
```

**Review by asjohnston-asf — APPROVED** [review]

**Comment by asjohnston-asf** [comment]
I confirmed the trufflehog failure is a false positive. It's flagging a git commit hash in one of the URLs in the updated documentation.

**Pull request merged**